corehq/apps/export/models/new.py

```diff
@@ -150,20 +150,25 @@ def create_default_from_export_item(group_schema_path, item, app_ids_and_version
         """
 
         is_main_table = group_schema_path == MAIN_TABLE
-        is_advanced = isinstance(item, SystemExportItem) and item.is_advanced
-        transform = item.transform if isinstance(item, SystemExportItem) else None
+        is_system_property = isinstance(item, SystemExportItem)
+        is_advanced = is_system_property and item.is_advanced
```

Comment:
man the more i look at this the more i feel like system properties don't belong in the schema, but just the export instance... sorry that was probably my bad